_data/simple-icons.json

```
@@ -3690,6 +3690,11 @@
             "hex": "262729",
             "source": "https://www.eightsleep.com/press/"
         },
+        {
+            "title": "EJS",
+            "hex": "A91E50",
+            "source": "https://ejs.co/favicon.svg"
```

```suggestion
            "source": "https://github.com/mde/ejs-site/blob/dd845093a46b373df42108b888a0ad80085a5c94/ejs.svg"
```

There's a higher resolution version of the logo on GitHub that we can use here - could you update using that as your source?